[OSDOCS-4241]: Add a note for cluster updates with various etcd member sizes

tmalove opened the pull request:
[OSDOCS-4241](https://issues.redhat.com/browse/OSDOCS-4241)

Version(s): 4.11+

Issue: [OCPBUGS-1756](https://issues.redhat.com/browse/OCPBUGS-1756)

Link to docs [preview](http://file.rdu.redhat.com/tlove/tlove-etcd-osdocs-4241/scalability_and_performance/recommended-host-practices.html#recommended-etcd-practices_recommended-host-practices). (Updated 10/24 at 9:16PM)


Additional information:
This card is a result of Slack discussion where a customer was concerned about doing a cluster upgrade with the etcd leader size different from the other members. It was found to be not-a-bug, however, the reporter wanted a note just to assure the reader that this is not a concern.

tmalove commented:
@vsolanki12 @geliu2016  Review this note in the 'Recommended etcd practices'. Thanks.

vsolanki12 commented:
thanks, @tmalove, Yes saw the note is added in the document. If we can add more details with logs than it will be fine.

geliu2016 commented:
/lgtm 
/label cherry-pick-approved

tmalove commented:
> thanks, @tmalove, Yes saw the note is added in the document. If we can add more details with logs than it will be fine.

Thanks for reviewing @vsolanki12 . I will look for a log snippet. Is there any particular detail that you are looking to add?

vsolanki12 commented:
@tmalove, is the note removed from the documentation? I am not able to find in the documentation. https://docs.openshift.com/container-platform/4.11/scalability_and_performance/recommended-host-practices.html#recommended-etcd-practices_recommended-host-practices.

tmalove commented:
Hi @vsolanki12  Let me know if we can merge this commit with the note and then I can open another PR to add log details afterward? If you agree, pls ACK and I will get QE to do the same. I will add you to the subsequent PR. Thanks!

tmalove commented:
/label peer-review-needed

GroceryBoyJr reviewed (commented):
/label LGTM

GroceryBoyJr commented:
/LGTM

GroceryBoyJr commented:
/remove-label peer-review-in-progress
/label peer-review-done

GroceryBoyJr commented:
/label merge-review-needed

GroceryBoyJr commented:
/remove-label peer-review-needed

bergerhoffer commented on modules/recommended-etcd-practices.adoc:
```diff
@@ -23,14 +23,15 @@ The following hard disk features provide optimal etcd performance:
 
 Some key metrics to monitor on a deployed {product-title} cluster are p99 of etcd disk write ahead log duration and the number of etcd leader changes. Use Prometheus to track these metrics.
 
+NOTE: The etcd member database sizes can vary in a cluster during normal operations. This difference does not affect cluster upgrades, even if the leader size is different from the other members.
```
Would you mind changing this to the proper note formatting?

```suggestion
[NOTE]
====
The etcd member database sizes can vary in a cluster during normal operations. This difference does not affect cluster upgrades, even if the leader size is different from the other members.
====

I can't remember if there is any actual problem with the other syntax, but our guidelines say to use this syntax (see https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#admonitions).
```

bergerhoffer reviewed (commented):
@tmalove not critical, but would you mind fixing the syntax of the admonition before we merge?

I'm removing the merge-review-needed label, but you can add it back and/or ping me when you've updated and I'll merge! Thanks!

tmalove commented:
@bergerhoffer I corrected the admonition formatting, generated another preview to show that nothing was affected by the note, and added the 'merge-request' label. Thanks!

tmalove commented:
/label merge-review-needed

bergerhoffer commented:
@tmalove Let me know if this is ready (I don't see the networking file anymore :tada: ) and I'll take a look in the morning!

bergerhoffer commented:
Will check this first thing in the a.m.!

/label merge-review-in-progress

bergerhoffer reviewed (commented):
Appreciate the update, LGTM, merging!

bergerhoffer commented:
/cherrypick enterprise-4.12

bergerhoffer commented:
/cherrypick enterprise-4.11
